court_ball_game/tests.py

```diff
@@ -0,0 +1,61 @@
+from .models import CourtBallGame
+from court.models import Court
+from player.models import BallGame
+from decimal import Decimal
+from django.db.utils import IntegrityError
+from django.core.exceptions import ValidationError
+import pytest
+
+
+@pytest.mark.django_db
+class TestCourtBallGameModel:
+
+    @pytest.fixture
+    def court_instance(self):
+        court = Court.objects.create(courtID=1, x=Decimal('11'), y=Decimal('22'),
+                                     city='TEST_CITY', neighborhood='TEST_NEIGHBORHOOD', max_players=30)
+        return court
+
+    @pytest.fixture
+    def court_ball_game_instance(self, court_instance):
+        court_ball_game = CourtBallGame.objects.create(court=court_instance, ball_game=BallGame.Basketball)
+        return court_ball_game
+
+    def test_unique_together(self, court_instance, court_ball_game_instance):
+        # CourtBallGame.objects.create(court=court_instance, ball_game=BallGame.Basketball)
+        with pytest.raises(IntegrityError):
+            CourtBallGame.objects.create(court=court_instance, ball_game=BallGame.Basketball)
```

Comment:
this way it will be easier to understand
```suggestion
            CourtBallGame.objects.create(court=court_ball_game_instance.court, ball_game=court_ball_game_instance.ball_game)
            
            
            
```

Reply:
Fixed